app/Jobs/CloneImagesOrVideos.php

```diff
@@ -134,7 +141,7 @@ public function handle()
                 }
             }
             if ($copy->files()->exists()) {
-                ProcessNewVolumeFiles::dispatch($copy);
+                ProcessNewVolumeFiles::dispatch($copy, [], $this->uuidMap);
```

Comment:
You can also use named arguments in cases like these:

```suggestion
                ProcessNewVolumeFiles::dispatch($copy, uuidMap: $this->uuidMap);
```

Reply:
@mzur 
When this change is applied I get this error: Biigle\Jobs\ProcessCloneVolumeFiles::__construct(): Argument #2 ($only) not passed

Reply:
Curious, I thought it would use the default value of `$only` then. You probably have to use the names of all arguments but this is even more verbose than before so forget it.